sdk/resourcemanager/Azure.ResourceManager/src/Common/Generated/ResourceManagerModelFactory.cs

```diff
@@ -1,19 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-// <auto-generated/>
-
-#nullable disable
-
-using System;
-using System.Collections.Generic;
-using Azure;
-using Azure.Core;
-
-namespace Azure.ResourceManager.Models
-{
-    /// <summary> Model factory for models. </summary>
-    public static partial class ResourceManagerModelFactory
-    {
-    }
```

Comment:
This is not a breaking change (CI is not reporting this) because we have other partial class definitions of this type